README.md

```diff
@@ -53,6 +53,25 @@ var data = [
 ];
 ```
 
+More advanced data objects can also be used but this requires us to specify the which attribute that contains the date we want to use. This object is available during the eventColor and eventClick callbacks for example.
```

Comment:
Not sure, but `to specify which attribute is the date` sounds better to me (but I'm not native English (-:)